### ISSUE TITLE
File types that require extra packages to import

### ISSUE THREAD

**Issue by dannyparsons** [IDEMSInternational/R-Instat#5182]
See the table here from `rio` https://github.com/leeper/rio on the packages that are installed by default. Many are not, such as `readODS`, which means you get an error trying to import `.ods` with `rio` until you install the `readODS` package yourself. This is to keep the `rio` package itself fairly small.

So we need to decide what we do with these formats for R-Instat. For each of the extra formats we could:
a) Install them by default in R-Instat so they are always available
b) Not install by default but have the packages download so they could be installed offline when needed.
c) Not install by default and don't have the package offline but have a way for users to easy download it online.
d) Remove the format from our list of files

a) and b) have implications on the installation process if more packages need to be installed and on the download size.

I think this should be a case-by-case basis depends on how important we feel the format is. I think we probably never want to do d) because it removes a possible format that could be imported with the right package installed. More common ones could be a) and less common could be b) or c).

For now I'd like to suggest we do a) for `.ods` files as they are important especially for our Linux community.

**Comment by rdstern** [comment]
I agree with a for ods files.  We needed them for the AIMS teaching.
I also notice that rio, in importing files a=can also read (and write) zip files.  This seems very interesting as a way of being able to read multiple files of a particular type.

I hope the open of multiple files and zip files does not affect the single sheets/files that we have now.  I realise this might be easy to have as options on our existing File > Open, but if it causes issues I wouldn't mind if it even became a separate dialogue.

I also notice it can open html files.

On writing, I notice it can now write multiple sheets of an Excel file.  I wonder if we should have a special menu item File > Export > Export to Excel.  This could be disabled unless we have chosen to add the extra Rcpp package when installing R-Instat.  I notice it can now also write multiple data frames - might this be useful if we want to export them in a way that can easily then be read into RStudio?

**Comment by dannyparsons** [comment]
`readODS` is now added.

I think probably doing c) for the remaining types would be a good at least for now. That's much better than getting giving the error that you currently get.
